src/core/computations/python/loadPython.ts

```diff
@@ -13,17 +13,17 @@ declare global {
 
 export async function loadPython() {
   window.pyodide = await window.loadPyodide({
-    indexURL: "https://cdn.jsdelivr.net/pyodide/v0.19.0/full/",
+    indexURL: "https://python.rijksoverhemd.nl",
```

Comment:
Revert this line to how we found it

```suggestion
    indexURL: "https://cdn.jsdelivr.net/pyodide/v0.19.0/full/",
```